statsmodels/tsa/holtwinters.py

```diff
@@ -0,0 +1,690 @@
+                                     dampen(b[i - 1], phi))) + (gammac * s[i - 1])
+            slope = b[:i].copy()
+            season = s[m:i + m].copy()
+            l[i:] = l[i]
+            b[:i] = dampen(b[:i], phi)
+            b[i:] = dampen(b[i], phi_h)
+            s[i + m - 1:] = [s[(i - 1) + j % m] for j in range(h + 1)]
+            fitted = trended(l, b) + s[:-m]
+        else:
+            for i in range(1, n + 1):
+                l[i] = y_alpha[i - 1] + \
+                    (alphac * trended(l[i - 1], dampen(b[i - 1], phi)))
+                if trending:
+                    b[i] = (beta * detrend(l[i], l[i - 1])) + \
+                        (betac * dampen(b[i - 1], phi))
+            slope = b[:i].copy()
+            season = s[m:i + m].copy()
+            l[i:] = l[i]
+            b[:i] = dampen(b[:i], phi)
+            b[i:] = dampen(b[i], phi_h)
+            fitted = trended(l, b)
+        level = l[:i].copy()
+        if use_boxcox or use_boxcox == 'log':
+            fitted = inv_boxcox(fitted, lamda)
+            # TODO: Does it make sense to have a inv_boxcox transform of the level and trend?
+            #level = inv_boxcox(level, lamda)
+            #slope = inv_boxcox(slope, lamda)
+            # TODO: How do we deal with the inv_boxcox transform on negative seasonal components?
+            #season = inv_boxcox(season, lamda)
+        SSE = sqeuclidean(fitted[:-h], data)
+        # (s0 + gamma) + (b0 + beta) + (l0 + alpha) + phi
+        k = m * seasoning + 2 * trending + 2 + 1 * damped
+        AIC = n * np.log(SSE / n) + (k) * 2
+        AICc = AIC + (2 * (k + 2) * (k + 3)) / (n - k - 3)
+        BIC = n * np.log(SSE / n) + (k) * np.log(n)
+        resid = data - fitted[:-h]
+        if remove_bias:
+            fitted += resid.mean()
+        if not damped:
+            phi = None
+        self.params = {'alpha': alpha,
+                       'beta': beta,
+                       'gamma': gamma,
+                       'phi': phi,
+                       'l0': l0,
+                       'b0': b0,
+                       's0': s0,
+                       'use_boxcox': use_boxcox,
+                       'lamda': lamda,
+                       'remove_bias': remove_bias}
+        hwfit = HoltWintersResults(self, self.params, fittedfcast=fitted, fittedvalues=fitted[:-h],
+                                   fcast=fitted[-h_fcast:], SSE=SSE, level=level,
+                                   slope=slope, season=season, AIC=AIC, BIC=BIC,
+                                   AICc=AICc, resid=resid, k=k)
+        return HoltWintersResultsWrapper(hwfit)
+
+
+class SimpleExpSmoothing(ExponentialSmoothing):
+    """
+    Simple Exponential Smoothing wrapper(...)
+
+    Parameters
+    ----------
+    endog : array-like
+        Time series
+
+    Returns
+    -------
+    results : SimpleExpSmoothing class
+
+    Notes
+    -----
+    This is a full implementation of the simple exponential smoothing as
+    per [1].
+
```

Comment:
Perhaps a 

```
See Also
---------
Exponential Smoothing
Holt
```

(not sure if you need full path or not, there are examples in SM)